docs/source/advancedcommands.rst

```diff
@@ -1,34 +1,12 @@
 Advanced Commands
 *****************
 
-Feasibility Analysis
+Relaxation Analysis
 ====================
 
 If your Model doesn't solve, you can automatically find the nearest feasible version of it with the ``Model.feasibility()`` command, as shown below. The feasible version can either involve relaxing all constraints by the smallest number possible (that is, dividing the less-than side of every constraint by the same number), relaxing each constraint by its own number and minimizing the product of those numbers, or changing each constant by the smallest total percentage possible.
```

Comment:
Fixed stale reference, saved your comment above to figure out what to do with later.